client/src/elements/schema-editor/schema-editor-files.js

```diff
@@ -1,11 +1,22 @@
 import { bindable, inject, Loader, LogManager } from "aurelia-framework";
+import { HttpClient } from "aurelia-fetch-client";
 import { Notification } from "aurelia-notification";
 import { I18N } from "aurelia-i18n";
 import qEnv from "resources/qEnv.js";
+import QConfig from "resources/QConfig";
 import { AuthService } from "aurelia-authentication";
 const log = LogManager.getLogger("Q");
 
-@inject(Loader, AuthService, Notification, I18N)
+function blobToDataURL(blob) {
```

Comment:
Didn't `URL.createObjectURL(blob)` work for you?

Reply:
of course, thanks for the hint.